File: add/codesnippet/CPP/m-system.runtime.remotin_13_1.cpp
   // Create a SoapHexBinary object.
   SoapHexBinary^ hexBinary = gcnew SoapHexBinary;
   hexBinary->Value = gcnew array<Byte>(5){
      2,3,5,7,11
   };
   Console::WriteLine( L"The SoapHexBinary object is {0}.", hexBinary );

File: add/codesnippet/CSharp/m-system.runtime.remotin_13_1.cs
        // Create a SoapHexBinary object.
        SoapHexBinary hexBinary = new SoapHexBinary();
        hexBinary.Value = new byte[]{ 2, 3, 5, 7, 11 };
        Console.WriteLine("The SoapHexBinary object is {0}.", 
            hexBinary.ToString());